pkgs/applications/networking/mailreaders/mutt/default.nix

```diff
@@ -29,9 +34,10 @@ stdenv.mkDerivation rec {
     sha256 = "1c8vv4anl555a03pbnwf8wnf0d8pcnd4p35y3q8f5ikkcflq76vl";
   };
 
-  patchPhase = optionalString (openssl != null) ''
-    sed -i 's#/usr/bin/openssl#${openssl}/bin/openssl#' smime_keys.pl
-  '';
+  patches = optional smimeSupport (fetchpatch {
+    url    = "https://sources.debian.net/src/mutt/1.7.2-1/debian/patches/misc/smime.rc.patch";
```

Comment:
This patch works with our version of mutt?

Reply:
Basically it enables the configuration samples, minus a few options. I tested it with GnuPG. Someone should try S/SMIME but I think it should be fine.